docs/dotnet-suggest.md

```diff
@@ -7,20 +7,28 @@ Command line apps built using `System.CommandLine` have built-in support for tab
 On the machine where you'd like to enable completion, you'll need to do two things.
 
 1. Install the `dotnet-suggest` global tool:
-   
-   [![Nuget](https://img.shields.io/nuget/v/dotnet-suggest.svg)](https://nuget.org/packages/dotnet-suggest)
-
-2. Add the appropriate shim script to your shell profile. You may have to create a shell profile file. The shim script will forward completion requests from your shell to the `dotnet-suggest` tool, which delegates to the appropriate `System.CommandLine`-based app.
-   
-   * For bash, add the contents of [dotnet-suggest-shim.bash](https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Suggest/dotnet-suggest-shim.bash) to `~/.bash_profile`.
-   
-   * For zsh, add the contents of [dotnet-suggest-shim.zsh](https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Suggest/dotnet-suggest-shim.zsh) to `~/.zshrc`.
-  
-   * For PowerShell, add the contents of [dotnet-suggest-shim.ps1](https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Suggest/dotnet-suggest-shim.ps1) to your PowerShell profile. You can find the expected path to your PowerShell profile by running the following in your console:
-
-```console
-> echo $profile
+
+```sh
+dotnet tool install dotnet-suggest
 ```
 
-(For other shells, please [look for](https://github.com/dotnet/command-line-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22shell+suggestion%22) or open an [issue](https://github.com/dotnet/command-line-api/issues).)
+2. Install the completion script.
 
+### bash
+```sh
+dotnet-suggest script bash >~/.dotnet-suggest-shim.bash
+echo '. ~/.dotnet-suggest-shim.bash' >>~/.bashrc
+```
+
+### zsh
+```sh
+dotnet-suggest script zsh >~/.dotnet-suggest-shim.zsh
+echo '. ~/.dotnet-suggest-shim.zsh' >>~/.zshrc
+```
+
+### PowerShell
+```sh
+?
```

Comment:
What would be the appropriate Windows commands?

Reply:
@KalleOlaviNiemitalo do you use Windows? Can you suggest something for this?

Reply:
Since our recommended approach is to copy the content of the shim script into your PowerShell profile, it seems to avoid the need to sign the script. I haven't heard of that being an issue.

Reply:
Thanks for that information. I've revered the changes for PowerShell.